test/GoodGhostingGasEstimate.test.js

```diff
@@ -43,7 +43,25 @@ contract("GoodGhostingGasEstimate", (accounts) => {
     describe("simulates a full game with 5 players and 4 of them winning the game and with admin fee % as 0", async () => {
         it("initializes contract instances and transfers DAI to players", async () => {
             token = new web3.eth.Contract(daiABI, providersConfigs.dai.address);
-            goodGhosting = await GoodGhostingArtifact.deployed();
+            // overriding deployment to test with 0% custom fees
+            if (process.env.NETWORK === "local-mainnet-fork") {
+                const poolConfigs = configs.providers[configs.deployConfigs.selectedProvider.toLowerCase()]['mainnet'];
+                const lendingPoolAddressProvider = poolConfigs.lendingPoolAddressProvider;
+                const dataProviderAddress = poolConfigs.dataProvider;
+                const inboundCurrencyAddress = poolConfigs[configs.deployConfigs.inboundCurrencySymbol.toLowerCase()].address;
+                goodGhosting = await GoodGhostingArtifact.new(inboundCurrencyAddress, lendingPoolAddressProvider, segmentCount, segmentLength, segmentPayment, earlyWithdrawFee, 0, dataProviderAddress, configs.deployConfigs.merkelroot);
+            } else {
+                const poolConfigs = configs.providers[configs.deployConfigs.selectedProvider.toLowerCase()]['polygon'];
+                const lendingPoolAddressProvider = poolConfigs.lendingPoolAddressProvider;
+                const incentiveController = poolConfigs.incentiveController;
+                const router = poolConfigs.router;
+                const usdc = poolConfigs.usdc;
+                const wmatic = poolConfigs.wmatic;
+                const dataProviderAddress = poolConfigs.dataProvider;
+                const inboundCurrencyAddress = poolConfigs[configs.deployConfigs.inboundCurrencySymbol.toLowerCase()].address;
+                goodGhosting = await GoodGhostingArtifact.new(inboundCurrencyAddress, lendingPoolAddressProvider, segmentCount, segmentLength, segmentPayment, earlyWithdrawFee, 0, dataProviderAddress, configs.deployConfigs.merkelroot, incentiveController, router, wmatic, usdc);
+            }
+            // goodGhosting = await GoodGhostingArtifact.deployed();
```

Comment:
yeah actually @fodisi  we did not had this code before we introduced the admin fee concept so in order to prevent testing 0 % fee by making a change everytime in the deploy config I did this but yeah if for testing we are okay to change the deploy config no worries I'll revert this